apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -63,19 +63,19 @@ Performance Limits:
 
 Disk throughput is capped at 19,000 Mbps and IOPS is capped at 80,000 IOPS for all compute add-ons.
 
-| Compute add-on | Max Disk Throughput | Baseline Disk Throughput | Max IOPS    | Baseline IOPS |
-| -------------- | ------------------- | ------------------------ | ----------- | ------------- |
-| Nano (free)    | 2,085 Mbps          | 43 Mbps                  | 11,800 IOPS | 250 IOPS      |
-| Micro          | 2,085 Mbps          | 87 Mbps                  | 11,800 IOPS | 500 IOPS      |
-| Small          | 2,085 Mbps          | 174 Mbps                 | 11,800 IOPS | 1,000 IOPS    |
-| Medium         | 2,085 Mbps          | 347 Mbps                 | 11,800 IOPS | 2,000 IOPS    |
-| Large          | 4,750 Mbps          | 630 Mbps                 | 20,000 IOPS | 3,600 IOPS    |
-| XL             | 4,750 Mbps          | 1,188 Mbps               | 20,000 IOPS | 6,000 IOPS    |
```

Comment:
These entries are correct - we're no longer clamping these numbers by the perf of the default gp3 disk.

I think a warranted change would be to make that clearer. It's mentioned [here](https://github.com/supabase/supabase/pull/29549/files#diff-f422479d6b0fb335255b465f9007f2381f13073f7252dffab763b0ad9535c2b5L55) but likely not very obvious to users.